index.js

```diff
@@ -138,47 +174,69 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
-
+let i = 0
+let a = 1
+function takeATicketNumber(line) {
+  line.push(a)
+  i++
+  a++
+  return `Welcome. You are ticketNumber${i}`
+}
```

Comment:
This works, but it can be simplified!

```suggestion
let i = 0
function takeATicketNumber(line) {
  i++
  line.push(i)
  return `Welcome. You are ticketNumber${i}`
}
```